lightning/src/ln/channelmanager.rs

```diff
@@ -12476,6 +12512,13 @@ impl_writeable_tlv_based_enum!(PendingHTLCRouting,
 		(9, payment_context, option),
 		(11, invoice_request, option),
 	},
+	(3, TrampolineForward) => {
+		(0, incoming_shared_secret, required),
+		(1, onion_packet, required),
+		(2, blinded, option),
+		(3, node_id, required),
+		(4, incoming_cltv_expiry, option),
```

Comment:
May as well make all of these even/"must understand" if we're adding a new variant